Header Rsimulation
uint32 left_pressed
uint32 right_pressed
uint32 sonar_value
uint32 light_value
